plugins/cue_points/base/lib/kCuePointManager.php

```diff
@@ -30,6 +32,19 @@ public function shouldConsumeDeletedEvent(BaseObject $object)
 	}
 	
 	/* (non-PHPdoc)
+	 * @see kObjectCreatedEventConsumer::shouldConsumeCreatedEvent()
+	*/
+	public function shouldConsumeCreatedEvent(BaseObject $object)
+	{
+		if($object instanceof mediaInfo)
```

Comment:
@tan-tan-kanarek The check requires retrieving both the flavorAsset and it's entry from the DB.
Even though they'll be cached in memory, working this way will require duplicating the code in copyCuePointsFromLiveToVodEntry().
That is the reason the code is part copyCuePointsFromLiveToVodEntry() in the first place.